stable/test/integration/_exec.pony

```diff
@@ -0,0 +1,102 @@
+use "ponytest"
+use "files"
+use "process"
+use "regex"
+
+actor _Exec
+  new create(
+    h: TestHelper,
+    cmdline: String,
+    notifier: ProcessNotify iso)
+  =>
+    try
+      let args = cmdline.split_by(" ")
+      let path = FilePath(h.env.root as AmbientAuth,
+        _env_with_default(h.env.vars, "STABLE_BIN", "stable"))?
```

Comment:
:x: realised that this "default" isn't helpful. I'll make it throw an error if no variable is found.